Library/Homebrew/cli/parser.rb

```diff
@@ -64,10 +64,17 @@ def switch(*names, description: nil, env: nil, required_for: nil, depends_on: ni
           set_constraints(name, required_for: required_for, depends_on: depends_on)
         end
 
-        enable_switch(*names, from: :env) if !env.nil? && !ENV["HOMEBREW_#{env.to_s.upcase}"].nil?
+        enable_switch(*names, from: :env) if env?(env)
       end
       alias switch_option switch
 
+      def env?(env)
+        return false if env.blank?
+        return true if env == :verbose && ENV["VERBOSE"]
```

Comment:
```suggestion
        return true if env == :verbose && ENV["VERBOSE"]
```

As discussed in person: let's just remove this (for now at least) because we filter it out in `bin/brew`.